runtime-builder/go-build.sh

```diff
@@ -30,8 +30,8 @@ export PATH=/usr/local/go/bin:"${PATH}"
 export GOPATH="${workspace}"/_gopath
 
 # Move application files into a temporary staging directory, dependencies excluded.
-staging=$(mktemp -d staging.XXXX)
-find . -mindepth 1 -maxdepth 1 ! -name "${staging}" ! -name "$(basename $GOPATH)" -exec mv {} "${staging}"/ \;
+staging="/staging"
+find . -mindepth 1 -maxdepth 1 ! -name "$(basename $GOPATH)" -exec mv {} "${staging}"/ \;
```

Comment:
Let's move the app files under ${staging}/app.  And instead of creating bin directory under ${workspace}, create it under ${staging}/bin instead.

In the future, we may need to also copy the debugger agent and launcher into ${staging}/bin.